dev-db/unixODBC/unixodbc-2.3.4.recipe

```diff
@@ -0,0 +1,95 @@
+SUMMARY="Open Source ODBC sub-system"
+DESCRIPTION="unixODBC is an Open Source ODBC sub-system and an ODBC SDK \
+for Linux, Mac OSX, and UNIX."
+HOMEPAGE="http://www.unixodbc.org/"
+COPYRIGHT="1999-2015 Peter Harvey
+	Nick Gorham"
+LICENSE="GNU GPL v2
+	GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="http://www.unixodbc.org/unixODBC-$portVersion.tar.gz"
+CHECKSUM_SHA256="2e1509a96bb18d248bf08ead0d74804957304ff7c6f8b2e5965309c632421e39"
+SOURCE_DIR="unixODBC-$portVersion"
+
+GLOBAL_WRITABLE_FILES="
+        settings/odbc.ini keep-old
+        settings/odbcinst.ini keep-old
+	"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	unixodbc${secondaryArchSuffix} = $portVersion
+	cmd:odbc_config${secondaryArchSuffix}
```

Comment:
Warning: POLICY ERROR: "/packaging/unixodbc_devel/bin/odbc_config" needs library "libroot.so", but the package doesn't seem to declare that as a requirement